r/tools/nixlibs.R

```diff
@@ -212,21 +212,30 @@ find_available_binary <- function(os) {
 download_source <- function() {
```

Comment:
It would probably be better to pass the global variable `VERSION` as the argument `version` in the call to `download_source()` and then refer to it here in the function as `version`. But maybe this PR isn't the place for that change.

Reply:
Yeah, I agree, though think a follow on is probably better for that (maybe along with a way to select a binary version  for developers?)